osu.Game/Graphics/DrawableDate.cs

```diff
@@ -58,8 +58,10 @@ private void updateTimeWithReschedule()
 
         public override bool HandleMouseInput => true;
 
-        private void updateTime() => Text = date.Humanize();
+        protected virtual string Format() => Text = date.Humanize();
```

Comment:
Return the string here, then when called in `updateTime` set `Text` to the return value.